### DISCUSSION TITLE
マイページの「ピックアップ記事」の歯車マークからチェックを外しても、ピックアップ記事が残ったままになる。

### DISCUSSION THREAD

**Discussion by Puchanko** [increments/qiita-discussions#623] [要望・アイデア・不具合]
初めまして。マイページにある「ピックアップ記事」を外したいので、歯車マークを押してチェックを外しても、マイページ上にピックアップ記事が残ったままになります。
どうすればピックアップ記事を外すことができますか？それとも不具合でしょうか？
ご確認宜しくお願いします。

**Comment by tanaka-asami415** [comment]
@Puchanko 
はじめまして！お問合せありがとうございます。

チェックを外された後、右下の「保存する」ボタンは押していただいているでしょうか？
弊社で確認したところ不具合はなさそうなため、一度ご確認いただけますと幸いです🙏

<img width="400" alt="image.png" src="https://github.com/increments/qiita-discussions/assets/103916185/ef5c6e9a-0d98-4621-ad95-4a5c938da7b3">

**Reply by Puchanko** [reply]
回答ありがとうございます。
少しスクロールしないと「保存する」が表示されない仕様だったのですね。。
それは気付きませんでした。
無事に解決はしました。ありがとうございます。
ただ、「保存する」ボタンがスクロールしなくても見える位置にあればいいなと個人的には思います。

**Reply by tanaka-asami415** [reply]
@Puchanko 
こちらこそご確認と詳細を教えていただきありがとうございます。

画面の高さが低い時に「保存する」ボタンが見えなくなってしまう仕様になっておりました…🙏
どんな画面の高さの時でも、ボタンが見えるように修正いたします。教えていただきありがとうございます！

**Reply by tanaka-asami415** [reply]
@Puchanko 
お待たせいたしました。「保存する」ボタンを常に見えるように修正しました！
https://qiita.com/release-notes/2024-03-12

この度はご報告いただきありがとうございました🙇✨

**Reply by Puchanko** [reply]
@tanaka-asami415 
早速のご対応ありがとうございます！
動作も問題ございませんでした！
今後のQiitaも楽しみです♪
